content/en/platform/corda/5.0-dev-preview-1/tutorials/run-demo-cordapp.md

```diff
@@ -79,43 +79,28 @@ The project containing the sample CorDapp opens.
 
 `corda-cli network config docker-compose solar-system`
 
-3. Deploy your network by building the network deployment Docker file using Corda CLI:
-  `corda-cli network deploy -n solar-system -f solar-system.yaml > solar-system-compose`
+3. Run a Gradle command ([or run a Gradle task in IntelliJ](https://www.jetbrains.com/help/idea/work-with-gradle-tasks.html#gradle_tasks)) to build the CorDapp and generate the `.cpb` file:
 
-    The `-n` here is the name of the network. The `-f` is the network definition file.
+  `./gradlew build`
 
-    After you run the command, you will see a new `docker-compose.yaml` file in your project. This contains all of the details for your network, which is now ready to go.
+This command builds the CorDapp package (`.cpk`) files. Most CorDapps will have two files: a `contracts` file and a `workflows` file. The command also builds the CorDapp package bundle (`.cpb`).
 
-4. Run a Gradle command to build the CorDapp and generate the `.cpk` files:
+4. Deploy the network using `corda-cli` and `docker-compose`:
+  `corda-cli network deploy -n solar-system | docker-compose -f - up`
```

Comment:
perhaps we can add an info tip saying that you can also output the contents to a file if you want to see what is happening. But mention that piping to docker is the safer preferred way. 

The command would be the same one as before `corda-cli network deploy -n solar-system -f solar-system.yaml > solar-system-compose.yaml`